--- anki-2.1.0beta25/qt/aqt/mediasrv.py
+++ anki-2.1.0beta25/qt/aqt/mediasrv.py
@@ -13,7 +13,7 @@
 
 
 def _getExportFolder():
-    data_folder = aqt_data_folder()
+    data_folder = os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "share", "anki")
     webInSrcFolder = os.path.abspath(os.path.join(data_folder, "web"))
     if os.path.exists(webInSrcFolder):
         return webInSrcFolder
